src/content/11/en/part11a.md

```diff
@@ -32,11 +54,23 @@ We'll discuss each of these steps (and when they're suitable) in more detail lat
 
 Usually strict definitions act as a constraint on creativity/development speed, this, however should usually not be true for CI. This strictness should be set up in such a way as to allow for easier development and working together. Using a good CI system (GitHub Actions is what we'll cover in this module) will allow us to do this all automagically.
 
+### Packaging and Deployment as a part of CI
+
+It may be worthwhile to note that packaging and especially deployment are sometimes considered to not fall under the umbrella of CI. We'll add them in here because in the real world it makes sense to lump it all together. This is partially because they make sense in the context of the flow and pipeline (I want to get my code to users) and partially because these are in fact the most likely point of failure.
+
+If a developer is working correctly in their own environment, it is unlikely that they will push code that would fail linting or tests. They would have already checked that on their own machine. The build too would likely pass because the tests would have likely had something to say about any issues that would prevent a build. It is possible that there will be a failure related to code in the master branch from other developer's changes and finding these issues is arguably the most important job of CI.
```

Comment:
I don't really follow this paragraph. You say that it's _unlikely_ broken code will be pushed, but the last sentence says it's the most important job (why, if it's unlikely) and mentions some "other developer".

Reply:
Fair point, let's see if I can fix that.

Reply:
@anna-osipova I removed that paragraph and made a few changes to the next one. Does it make more sense now?